AZ3166/src/libraries/AudioV2/src/AudioClassV2.h

```diff
@@ -164,6 +164,46 @@ class AudioClass {
          */
         bool setVolume(uint8_t volume);
 
+        /**
+         * @brief   Read the given nau88c10 register
+         * 
+         * @param  registerAddress       Register address.
+         * 
+         * @returns the integer value in the register.
+         */        
+        uint16_t readRegister(uint16_t registerAddress);
```

Comment:
>readRegister [](start = 17, length = 12)

These new APIs are designed for advanced audio developers who want to control the audio codec from lower hardware level.
They are expected to deeply understand the codec primitives, such as PGA, ALC, etc, or have to refer to the NAU88C10 data sheet.